### PR TITLE
Temporarily undo reparsing machete file when repo changed

### PR DESCRIPTION
**Pull request by PawelLipski**
`Temporarily` = until next release or so, hopefully

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,24 +7,24 @@
 
 <img src="docs/logo.svg" style="width: 100%; display: block; margin-bottom: 10pt;"/>
 
-Git Machete plugin is a robust tool that **simplifies your git related workflow**.
+💪 Git Machete plugin is a robust tool that **simplifies your git related workflow**.
```
Comment (micpiotrowski) [inline README.md:10]: :heart:

**Review by micpiotrowski — COMMENTED** [review]
Is `machete` file reparsing still not working correctly?

**Comment by PawelLipski** [comment]
> Is `machete` file reparsing still not working correctly?

yes, pls see #679 :/ no idea why it still happens despite your fix in #675 ... I suspect a bug in IDEA itself

File: README.md [hunk 2]
```diff
@@ -7,24 +7,24 @@
```
Comment (mkondratek) [inline README.md:7]: while we are here.. what about mentioning [the medium blog post](https://medium.com/virtuslab/take-a-look-at-your-repository-from-a-new-perspective-with-git-machete-plugin-e2e5e4197a0b) under the *References* section?

File: scripts/validate-markdown-values [hunk 3]
```diff
@@ -9,7 +9,7 @@ source "$self_dir"/utils.sh
 # is up to date with the source of truth (Gradle config).
 
 expected=$(./gradlew -q resolveIntelliJVersions --versionKey=earliestSupportedMajor)
-actual=$(grep -Po '(?<=minimum required version is )[0-9.]+' README.md)
```
Comment (mkondratek) [inline scripts/validate-markdown-values:12]: whoops

**Review by mkondratek — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (PawelLipski) [reply README.md:7]: Done

**Pull request merged**